github/libp2p.yml

```diff
@@ -4308,6 +4308,7 @@ repositories:
             - Compile on x86_64-pc-windows-msvc
             - Compile with select features (mdns tcp dns async-std)
             - Compile with select features (mdns tcp dns tokio)
+            - Interoperability Testing
```

Comment:
This is the name of the workflow, not the name of the job.

Your workflow currently has two jobs but only the latter is likely the one we want to make required.

In the following screenshot, you can see the `Compile on wasm32-unknown-unknown` job that is mentioned on line 4305 above. There is no mention which workflow it is in (i.e. the `Continuous integration` isn't mentioned in here).

![image](https://user-images.githubusercontent.com/5486389/213613223-a59ceb9e-17e4-45f4-9356-65de82df7db4.png)

Interestingly enough, the actual test job has another level of nesting: https://github.com/libp2p/rust-libp2p/actions/runs/3962950188/jobs/6791796862

My guess would be that we should be changing this to:

```suggestion
            - Run testplans test
```

Reply:
Pff, thanks for this Thomas! :pray: Updated